tests/exporter_test.go

```diff
@@ -106,9 +76,8 @@ var _ = Describe("Exporter", func() {
 		ctx = context.Background()
 
 		//  Run the collector container
-
-		image, err := getCollectorImageFromDockerfile()
-		Expect(err).ToNot(HaveOccurred())
+		// renovate: datasource=docker
+		image := "otel/opentelemetry-collector-contrib:0.106.1"
```

Comment:
I bumped this dep down just to test that this is detected by renovate correctly.